### PR TITLE
Routing

### LINKED ISSUE #81: Add routes to the seed
The next step for this involves adding react-router structure, examples and tests, so the seed can grow happy beautiful routed trees.

### PR DESCRIPTION
**Pull request by renanrodrigo**
Add react-router, Route examples for components, tests for routes.

Fix #81

### CODE REVIEW

**Comment by renanrodrigo** [comment]
Yes, I know, dependabot. WIll rebase as soon as #82 is merged or discarded.

**Comment by StykMartin** [comment]
> Yes, I know, dependabot. WIll rebase as soon as #82 is merged or discarded.

I don't see any rebase here :eyes:

**Comment by renanrodrigo** [comment]
slo-mo rebase done

File: package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
   "dependencies": {
     "@patternfly/react-core": "^4.23.1",
     "@types/react": "^16.9.43",
+    "@patternfly/react-icons": "^4.4.2",
```
Comment (StykMartin) [inline package.json:74]: Are you installing whole internets

Comment (renanrodrigo) [reply package.json:74]: Pattern Flying

**Pull request merged**